next-app/src/routes/projects/[project_code]/meta/+server.ts

```diff
@@ -6,7 +6,7 @@ export async function get({ project_code, cookie }) {
 		{ entries, comments }
 	] = await Promise.all([
 		sf({ name: 'project_read_by_code', args: [ project_code ], cookie }),
-		sf({ name: 'lex_stats', args: [ project_code ], cookie }),
```

Comment:
just curious, are you still using lex_stats in addition to this?  Or are you only using lex_stats_all now?  If it's no longer used, how about we remove it from PHP.

Reply:
I do not use lex_stats but it might be used in the PHP code... if you'd like I'll go digging around to make sure